pytest_selenium/pytest_selenium.py

```diff
@@ -64,6 +64,8 @@ def driver_kwargs(request, capabilities, driver_class, driver_path,
         host=request.config.getoption('host'),
         port=request.config.getoption('port'),
         request=request,
+        log_path=os.path.realpath('geckodriver.log'),
```

Comment:
Ahh.. so this is where `log_path` is being set.. This is Firefox specific so it shouldn't be set here. We *could* have a `driver_log` fixture, but I don't think we need one at this point.